internal/mage/engine.go

```diff
@@ -198,18 +163,49 @@ func (t Engine) test(ctx context.Context, race bool) error {
 
 	c = c.Pipeline("engine").Pipeline("test")
 
+	opts := util.DevEngineOpts{
+		ConfigEntries: map[string]string{
+			`registry."registry:5000"`:        "http = true",
+			`registry."privateregistry:5000"`: "http = true",
+		},
```

Comment:
I like the flexibility/expressiveness here

Reply:
Thanks, @vito!